### PR TITLE
Wk 169 upload document

### PR DESCRIPTION
**Pull request by aditudorache**
This PR adds a file input contol for uploading the documents.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "@babel/polyfill": "^7.7.0",
     "@datapunt/asc-assets": "^0.17.7",
     "@datapunt/asc-core": "^0.17.7",
-    "@datapunt/asc-ui": "^0.17.7",
+    "@datapunt/asc-ui": "^0.17.8-alpha.12",
```
Comment (janjaap) [inline package.json]: This is the canary version. Why would you upgrade to this version if the latest stable is still `0.17.7`?

File: src/views/manage/ManageForm.jsx [hunk 2]
```diff
@@ -58,8 +62,10 @@ const normalize = item => {
       eind_uitvoering,
       jaar_oplevering,
       notes,
+      documents,
     },
   } = item;
+  console.log(documents);
```
Comment (janjaap) [inline src/views/manage/ManageForm.jsx]: Forgot to remove

File: src/views/manage/ManageForm.jsx [hunk 3]
```diff
@@ -96,14 +104,17 @@ const ManageForm = ({ id }) => {
 
   useEffect(() => {
     Object.keys(initalValues).map(name => register({ name: name }));
+    ['rapport_document', 'design_document'].map(name =>
+      register({ name: name })
```
Comment (janjaap) [inline src/views/manage/ManageForm.jsx]: Or `register({ name })`

File: src/views/manage/ManageForm.jsx [hunk 4]
```diff
@@ -96,14 +104,17 @@ const ManageForm = ({ id }) => {
 
   useEffect(() => {
     Object.keys(initalValues).map(name => register({ name: name }));
+    ['rapport_document', 'design_document'].map(name =>
```
Comment (janjaap) [inline src/views/manage/ManageForm.jsx]: The `map` result isn't stored in a variable.

File: src/views/manage/ManageForm.jsx [hunk 5]
```diff
@@ -138,20 +149,47 @@ const ManageForm = ({ id }) => {
...
           </ControlsColumn>
-        </Row>
-        <Row>
+          <ControlsColumn
+            span={{ small: 1, medium: 2, big: 6, large: 6, xLarge: 6 }}
+          >
+            <Heading $as="h3" color="secondary">
+              Documenten
+            </Heading>
+            <FileInput
```
Comment (janjaap) [inline src/views/manage/ManageForm.jsx]: Should be a self-closing component; doesn't have children.

File: src/views/manage/ManageForm.jsx [hunk 6]
```diff
@@ -138,20 +149,47 @@ const ManageForm = ({ id }) => {
...
+              label="Rapportage"
+              name="rapport_document"
+              onChange={handleChange}
+              defaultValue={defaultValues['rapport_document']}
+            ></FileInput>
+            <FileInput
```
Comment (janjaap) [inline src/views/manage/ManageForm.jsx]: Same

File: src/views/manage/components/DatePickerInput.jsx [hunk 7]
```diff
@@ -69,4 +70,10 @@ const DatePickerInput = ({ name, onChange, label, defaultValue }) => {
   );
 };
 
+DatePickerInput.propTypes = {
+  name: PropTypes.string.isRequired,
+  onChange: PropTypes.func.isRequired,
+  defaultValue: PropTypes.shape({}).isRequired,
```
Comment (janjaap) [inline src/views/manage/components/DatePickerInput.jsx]: `label` prop type is missing

File: package.json (hunk 1 above)
Comment (aditudorache) [reply package.json]: I tryed to use a control from that version, but not anymore. I'll change it to the stabile version.

File: src/views/manage/ManageForm.jsx (hunk 2 above)
Comment (aditudorache) [reply src/views/manage/ManageForm.jsx]: thanks!

File: src/views/manage/ManageForm.jsx (hunk 4 above)
Comment (aditudorache) [reply src/views/manage/ManageForm.jsx]: not needed.

File: src/views/manage/ManageForm.jsx (hunk 5 above)
Comment (aditudorache) [reply src/views/manage/ManageForm.jsx]: ok!

File: src/views/manage/ManageForm.jsx (hunk 6 above)
Comment (aditudorache) [reply src/views/manage/ManageForm.jsx]: done!

File: src/views/manage/components/DatePickerInput.jsx (hunk 7 above)
Comment (aditudorache) [reply src/views/manage/components/DatePickerInput.jsx]: `label` is not used here anymore

Comment (janjaap) [reply src/views/manage/components/DatePickerInput.jsx]: 🤦‍♂ 
Missed that

**Pull request merged**